src/test/java/uk/gov/companieshouse/transactions/web/controller/confirmation/ConfirmationControllerTests.java

```diff
@@ -80,6 +80,40 @@ void getConfirmationSuccess() throws Exception {
                 .andExpect(model().attributeExists(TEMPLATE_NAME_ATTR));
     }
 
+    @Test
+    @DisplayName("Get confirmation view - Payment Cancelled Path")
+    void getPaymentCancelled() throws Exception {
+
+        Transaction closedTransaction = new Transaction();
+
+        when(transactionsService.getTransaction(TRANSACTION_ID)).thenReturn(closedTransaction);
+
+        this.mockMvc.perform(get(CONFIRMATION_PATH)
+        .param("status", "cancelled"))
+            .andExpect(status().is3xxRedirection());
```

Comment:
We could assert `view().name(resumeJourneyUri)` here too

Reply:
made suggested changes